octoprint_mrbeam/static/js/design_store.js

```diff
@@ -124,7 +124,10 @@ $(function () {
                 email: self.getEmail(),
                 serial: MRBEAM_SERIAL,
                 user_token: self.getAuthToken(),
-                version: BEAMOS_VERSION,
+                // TODO: remove the following sanitization once the version
```

Comment:
Please add the issue ID here in the comment for the issue that introduced the change and the issue ID for the one that should revert it

Reply:
Done